data/widgets/card/list.ui

```diff
@@ -92,6 +92,18 @@
                 </style>
               </object>
             </child>
+            <child>
+              <object class="GtkFrame" id="context-frame">
```

Comment:
fair enough, we just need to keep in mind that the looks of a GtkFrame might change, and if all we want is a placeholder, its usually better to use a box or a grid